src/client/core/utils/array.js

```diff
@@ -0,0 +1,71 @@
+export function toArray (obj) {
+    return Array.prototype.slice.call(obj);
+}
+
+export function isArray (obj) {
+    return obj instanceof Array;
```

Comment:
Use this:
```js
return Object.prototype.toString.call(arg) === '[object Array]';
```
Because instanceof only checks if Array.prototype is on an object's [[Prototype]] chain. It fails when checking arrays across frames since the Array constructor used for the instance will likely be different to the one used for the test.